### ISSUE TITLE
Nuxt does not throw error on browser when type checking error / fail

### ISSUE THREAD

**Issue by MYD07Shonet** [nuxt/typescript#283]
**Describe the bug**
When using Nuxt with typescript for the first time, there will obviously be some type checking fail but Nuxt only throw the error on the **terminal** not on browser or browser's console. If i use vanilla Nuxt every error will be listed on the browser so i am aware there are some errors need to be fixed.

**Expected behavior**
I expect the browser to throw error so every developer can easily aware of the error.

**Comment by kevinmarrec** [comment]
Hi @MYD07Shonet , The type checking is done on a separate process thanks to `https://github.com/TypeStrong/fork-ts-checker-webpack-plugin`.

You can try these 2 solutions :

# 1. Use ForkTsCheckerWebpackPlugin in `sync` mode (`async: false`)

```js
// nuxt.config
export default {
  typescript: {
    typeCheck: {
      async: false
    }
  }
}
```

# 2. Use `ts-loader` type check instead

```js
export default {
  typescript: {
    typeCheck: false, // disable ForkTsCheckerWebpackPlugin type checking
    loaders: {
      ts: {
        transpileOnly: false // enabled ts-loader type checking
      }
    }
  }
}
```

https://typescript.nuxtjs.org/guide/setup.html#typecheck
(https://typescript.nuxtjs.org/guide/setup.html#loaders)

---

If it doesn't do what you're looking for, I'm afraid there aren't workarounds.

**Comment by MYD07Shonet** [comment]
Dude, you rock brother. I have tried several examples on internet only your solution work. Thanks a lot

**Comment by kevinmarrec** [comment]
👌

**Comment by MYD07Shonet** [comment]
Hi @kevinmarrec the solution you give me using ts-loader transpileOnly: false working great with type checking and error throw, but as the project grow the compile time become noticeably slow. i am trying to use ForkTsCheckerWebpackPlugin ( which has better compile speed than th-loader ) with your config, but it doesn't do the type checking and don't throw any error which is very important. Is there any config that i miss somewhere to activate the same behaviour that ts-loader give ?
